examples/convert-legacy-llama.py

```diff
@@ -349,11 +349,13 @@ def load(model_plus: ModelPlus) -> Params:
 @dataclass
 class Metadata:
```

Comment:
Yeah. I agree with @Galunid on the dup here. Not sure how to dedup this though. Separate module seems like overkill?